src/main/java/de/konfidas/ttc/reporting/ReportTextPrinter.java

```diff
@@ -0,0 +1,26 @@
+package de.konfidas.ttc.reporting;
+
+import de.konfidas.ttc.messages.LogMessage;
+import de.konfidas.ttc.messages.TransactionLogMessage;
+import org.apache.commons.codec.binary.Hex;
+import org.apache.commons.lang3.StringUtils;
+import org.bouncycastle.asn1.ASN1Primitive;
+
+public class ReportTextPrinter {
```

Comment:
I think, we should not split up the report, the reporter and the printer of the report? so we have as input a

Collection<LogMessages> and Collection<ValidationException>, next, it is the reporters responsibility to do something with this, for example, create one XML, where all comes together... or some junit-like output. But I do not see the need for an intermediate "report", which can than again be printed into different formats. I guesss you had some reason to do so, could you explain what I am missing?